integration-tests/smoke/keeper_test.go

```diff
@@ -638,6 +644,77 @@ var _ = Describe("Keeper Suite @keeper", func() {
+			for _, nodeToTakeDown := range firstHalfToTakeDown {
+				err := nodeToTakeDown.DeleteJob("1")
+				Expect(err).ShouldNot(HaveOccurred(), "Could not delete the job from one of the nodes")
+				err = chainClient.WaitForEvents()
+				Expect(err).ShouldNot(HaveOccurred(), "Error deleting the Keeper job from the node")
+			}
+			log.Info().Msg("Successfully managed to take down the first half of the nodes")
+
+			// Assert that upkeeps are still performed and their counters have increased
+			Eventually(func(g Gomega) {
+				for i := 0; i < len(upkeepIDs); i++ {
+					currentCounter, err := consumers[i].Counter(context.Background())
+					g.Expect(err).ShouldNot(HaveOccurred(), "Failed to retrieve consumer counter"+
+						" for upkeep at index "+strconv.Itoa(i))
+					g.Expect(currentCounter.Int64()).Should(BeNumerically(">", initialCounters[i].Int64()),
+						"Expected counter to have increased from initial value of %s, but got %s",
+						initialCounters[i], currentCounter)
+				}
+			}, "3m", "1s").Should(Succeed())
+
+			// Take down the other half of the Keeper nodes
+			secondHalfToTakeDown := chainlinkNodes[len(chainlinkNodes)/2+1:]
+			for _, nodeToTakeDown := range secondHalfToTakeDown {
+				err := nodeToTakeDown.DeleteJob("1")
+				Expect(err).ShouldNot(HaveOccurred(), "Could not delete the job from one of the nodes")
+				err = chainClient.WaitForEvents()
+				Expect(err).ShouldNot(HaveOccurred(), "Error deleting the Keeper job from the node")
+			}
+			log.Info().Msg("Successfully managed to take down the second half of the nodes")
+
+			// See how many times each upkeep was executed
+			var countersAfterNoMoreNodes = make([]*big.Int, len(upkeepIDs))
+			for i := 0; i < len(upkeepIDs); i++ {
+				countersAfterNoMoreNodes[i], err = consumers[i].Counter(context.Background())
+				Expect(err).ShouldNot(HaveOccurred(), "Failed to retrieve consumer counter for upkeep at index "+strconv.Itoa(i))
+				log.Info().Msg("Upkeep at index " + strconv.Itoa(i) + " performed " +
+					strconv.Itoa(int(countersAfterNoMoreNodes[i].Int64())) + " times")
+			}
+
+			// Once all the nodes are taken down, there might be some straggling transactions which went through before
+			// all the nodes were taken down. Every keeper node can have at most 1 straggling transaction per upkeep,
+			// so a +6 on the upper limit side should be sufficient.
+			Consistently(func(g Gomega) {
+				for i := 0; i < len(upkeepIDs); i++ {
+					latestCounter, err := consumers[i].Counter(context.Background())
+					g.Expect(err).ShouldNot(HaveOccurred(), "Failed to retrieve consumer counter for upkeep at index "+strconv.Itoa(i))
+					g.Expect(latestCounter.Int64()).Should(BeNumerically("<=", countersAfterNoMoreNodes[i].Int64()+numUpkeepsAllowedForStragglingTxs),
+						"Expected consumer counter to not have increased more than %d, but got %d",
+						countersAfterNoMoreNodes[i].Int64()+6, latestCounter.Int64())
```

Comment:
nit. use numUpkeepsAllowedForStragglingTxs here

Reply:
Ahh sorry that I missed this...thanks for the heads-up!